src/Documentation/Grain-Versioning/Azure-deployment.md

```diff
@@ -0,0 +1,49 @@
+# Deploy new versions in Azure
+
+## Using Cloud Services
+
+### Rolling upgrade
+
+In this method you deploy a new cspkg directly on your Production environment.
+This is the simplest method, but it can be difficult to interrupt an on-going deployment
+and to rollback.
+
+Recommended configuration:
+- `DefaultCompatibilityStrategy` set to `BackwardCompatible`
+- `DefaultVersionSelectorStrategy` set to `AllCompatibleVersions`
+
+When using this configuration, "old" clients will be able to talk to activations
+on both versions of silos. Newer clients and silos will only trigger new activations
+on newer silos.
+
+![Rolling gif](rolling.gif)
+
+### Using a staging environment
+
+In this method you deploy a new cspkg in the Staging environment before doing
```

Comment:
The title of the pahe is "Deploy new versions in Azure" it was not intended to be generic. Should I do instead a generic page?